### ISSUE TITLE
Citations and license are outputted to terminal with the help message

### ISSUE THREAD

**Issue by KPLauritzen** [charnley/rmsd#18]
There is no reason to print these messages when the user ask for the help message
```
license:
  https://github.com/charnley/rmsd/blob/master/LICENSE

citation:
  Kabsch algorithm:
    Kabsch W., 1976, A solution for the best rotation to relate two sets of
    vectors, Acta Crystallographica, A32:922-923, doi:10.1107/S0567739476001873

  Quaternion algorithm:
    Michael W. Walker and Lejun Shao and Richard A. Volz, 1991, Estimating 3-D
    location parameters using dual number quaternions, CVGIP: Image
    Understanding, 54:358-367, doi: 10.1016/1049-9660(91)90036-o

  Implementation:
    Calculate RMSD for two XYZ structures, GitHub,
    http://github.com/charnley/rmsd
```

This is fine in the github readme, but when using it in terminal we only need the Usage information

**Comment by charnley** [comment]
I see your point. Will add reference til the GitHub page and delete all the cite/license info from --help.

**Comment by charnley** [comment]
Fixed in c5c7bd5de3db34ef2a5996f0b64dd37c72e1c298

**Comment by larsbratholm** [comment]
Could add "--licence" argument.
